.github/workflows/smoke-test.yml

```diff
@@ -68,32 +71,53 @@ jobs:
         run: go mod download
       - name: Build Binaries
         run: make
-      - name: Ensure bootpd is enabled
-        if: contains(matrix.os, 'macos')
+      - name: Ensure bootpd is enabled (macos)
+        if: matrix.os == 'macos-13'
         shell: bash
         run: |
           set -x
           fw=/usr/libexec/ApplicationFirewall/socketfilterfw
           sudo $fw --remove /usr/libexec/bootpd
           sudo $fw --add /usr/libexec/bootpd
           sudo $fw --unblock /usr/libexec/bootpd
-      - name: brew update
-        if: contains(matrix.os, 'macos')
-        run:
-          brew update
+      - name: Update brew (macos)
+        if: matrix.os == 'macos-13'
+        run: brew update
+      - name: Install tools (macos)
+        if: matrix.os == 'macos-13'
+        run: brew install tree
+      - name: Install vfkit and vmnet-helper (macos)
+        if: matrix.driver == 'vfkit'
+        run: |
+          brew install vfkit
+          machine="$(uname -m)"
+          archive="vmnet-helper-$machine.tar.gz"
+          curl -LOf "https://github.com/nirs/vmnet-helper/releases/latest/download/$archive"
+          sudo tar xvf "$archive" -C / opt/vmnet-helper
+          sudo install -m 0640 /opt/vmnet-helper/share/doc/vmnet-helper/sudoers.d/vmnet-helper /etc/sudoers.d/
       - name: Install qemu and socket_vmnet (macos)
-        if: contains(matrix.os, 'macos') && matrix.driver == 'qemu'
+        if: matrix.os == 'macos-13' && matrix.driver == 'qemu'
         run: |
           brew install qemu socket_vmnet
-          HOMEBREW=$(which brew) && sudo ${HOMEBREW} services start socket_vmnet          
+          HOMEBREW=$(which brew) && sudo ${HOMEBREW} services start socket_vmnet
       - name: Minikube Download only (iso/image)
         run:
-          ./out/minikube start --download-only --driver=${{ matrix.driver }} 
+          ./out/minikube start --download-only --driver ${{ matrix.driver }}
```

Comment:
lets add -v=8 to all minikube commands that will include the lib-machine super vebbose machine logs too